zerver/migrations/0353_userpresencenew_fill_data.py

```diff
@@ -63,6 +63,8 @@ class Migration(migrations.Migration):
     Ports data from the UserPresence model into the new one.
     """
 
+    atomic = False
```

Comment:
I think we want this, to really have batching in that `bulk_update` call, right?

Reply:
Yeah, that seems correct.  I guess the downside of doing that is that new UserPresence objects might be created in the meantime; we probably can just ignore that race since a normal client submits presence data every minute (and disabling the queue worker as discussed above would prevent such a race).

Reply:
Hmm, I don't think `atomic` a
influences this issue at all?

Reply:
No -- sorry I'd been thinking about switching this to a different loop structure that we write ourselves that could catch objects created while the migration is running.